spec/utility_methods_spec.rb

```diff
@@ -0,0 +1,245 @@
+require "spec_helper"
+
+RSpec.describe PgHaMigrations::SafeStatements, "utility methods" do
```

Comment:
Does `safe_set_maintenance_work_mem_gb` fall into this category as well?

Reply:
Yes, it does, I'll move that also.

Reply:
Done.